pkg/controller/middleware/sessions.go

```diff
@@ -30,24 +32,25 @@ import (
 
 const (
 	// sessionName is the name of the session.
-	sessionName = "verification-server-session"
+	sessionName     = "verification-server-session"
```

Comment:
Since these are included in the max length, should we pick something shorter like `_vs` and `_vsu`? That would force a logout on deploy, but I'm not super worried about that.

Reply:
I left it as is so that logged in users don't get logged out.